content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Application server configuration
+linkTitle: App server config
+description: Learn how to define agent paths for Java application servers
+weight: 15
+cSpell:ignore: asadmin binsetenv binstart Glassfish Payara setenv
+---
+
+When instrumenting an app that runs on a Java application server with a Java
+agent, you must add the `javaagent` path to the JVM arguments. The way to do
+this differs from server to server.
+
+## JBoss EAP / WildFly
+
+You can define the path to the Java agent by adding the `javaagent` argument to
+the `JAVA_TOOL_OPTIONS` environment variable. For example:
+
+{{< tabpane text=true persist=lang >}}
+
+{{% tab header="Linux" lang=Linux %}}
+
+```sh
+export JAVA_TOOL_OPTIONS="-javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{% tab header="Windows" lang=Windows %}}
+
+```bat
+set JAVA_TOOL_OPTIONS="-javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{< /tabpane >}}
```

Comment:
See https://github.com/open-telemetry/opentelemetry.io/pull/3459/files#r1396086080
Using `JAVA_TOOL_OPTIONS` can have unintended side effects. I think it is better to use the `JAVA_OPTS` way that follows. If needed using `JAVA_TOOL_OPTIONS` can be in a separate section (that is not related to wildfly)

Reply:
@laurit Would this do?

```suggestion
You can define the path to the Java agent by adding the `javaagent` argument to
the `JAVA_OPTS` environment variable. For example:

{{< tabpane text=true persist=lang >}}

{{% tab header="Linux" lang=Linux %}}

```sh
export JAVA_OPTS="-javaagent:/path/to/opentelemetry-javaagent.jar"
```

{{% /tab %}} {{% tab header="Windows" lang=Windows %}}

```bat
set JAVA_OPTS="-javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
```

{{% /tab %}} {{< /tabpane >}}
```

Reply:
Unfortunately no. The instructions ask user to add this at the end of the `standalone.conf` because in https://github.com/mojavelinux/jboss-as/blob/1eeb03a3fa0e795980a6ed96473df3ad0115902b/build/src/main/resources/bin/standalone.conf#L40 there is a test to check whether `JAVA_OPTS` is set, if not it adds some default arguments. When you set `JAVA_OPTS` outside the script setting the default options will be skipped. Unless you also want to explain that this is going to happen I'd suggest not including this in the documentation.

Reply:
I've updated the PR to just leave the standalone method. PTAL